runtime/queries/htmldjango/indents.scm

```diff
@@ -0,0 +1,2 @@
+(paired_statement) @indent
+(end_paired_statement) @outdent
```

Comment:
Thinking about this again, not sure if we want this since vim does not indent django paired statements.